dockstore-webservice/src/main/java/io/dockstore/webservice/resources/SourceControlResourceInterface.java

```diff
@@ -51,16 +51,34 @@
     default Token refreshBitbucketToken(Token token, HttpClient client, TokenDAO tokenDAO, String bitbucketClientID,
         String bitbucketClientSecret) {
 
-        String url = BITBUCKET_URL + "site/oauth2/access_token";
+        String refreshUrl = BITBUCKET_URL + "site/oauth2/access_token";
+        String payload = "grant_type=refresh_token&refresh_token=" + token.getRefreshToken();
```

Comment:
I wonder if there's a prettier way of generating the payload than this

Reply:
Not sure; I did what  was done before.